CMakeLists.txt

```diff
@@ -27,6 +27,18 @@ list( APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake )
 project( nest CXX C )
 set( NEST_USER_EMAIL "users@nest-simulator.org" )
 
+# this include for coloring messages
+include( Utils )
```

Comment:
Maybe rename the file to `ColorMessages.cmake` and remove the comment?

Please also add a new line between this `include` and the next block of code.